presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -95,6 +95,7 @@
     TOO_MANY_TASK_FAILED(0x0001_001B, INTERNAL_ERROR),
     INDEX_LOADER_TIMEOUT(0x0001_001C, INTERNAL_ERROR),
     EXCEEDED_TASK_UPDATE_SIZE_LIMIT(0x0001_001D, INTERNAL_ERROR),
+    NODE_SELECTION_STATEGY_NOT_SUPPORTED(0x0001_001E, INTERNAL_ERROR),
```

Comment:
Let's remove this. I don't think we should fail the query simply because node selection strategy is not supported.